Wordle.API/Wordle.API/Models/Score.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Wordle.Api.Models
+{
+    public record Score
```

Comment:
```suggestion
    public record PlayerDto
```
Based on our lecture fro today and from what we about dtos, I think thsi could be become a `PlayerDto` instead of Score. Since Score is just the Player Entity without the `PlayerId`. I also assume you will want to add in `AverageSecondsPerGame` once that feature is finished. This would also mean refactoring it into a `dtos` folder